frontend/demo/src/actions/tags/index.tsx

```diff
@@ -0,0 +1,113 @@
+import {createAction} from 'typesafe-actions';
```

Comment:
This seems unused, but I thought we were using `typesafe-actions` in the demo app.